src/components/shared/FilterNavigatorBar.css

```diff
@@ -125,25 +132,30 @@
 
 /* Animation */
 
+.filterNavigatorBarUncommittedTransition-exit {
+  /* Because of the underlying transition library, this element is still here
+   * while the new "committed" element is created, which pushes it further
+   * right. By using display: none here, we prevent this bad effect. */
+  display: none;
+}
+
 .filterNavigatorBarTransition-enter {
-  z-index: 0;
-  opacity: 0.1;
-  transform: translateX(-100%);
+  color: inherit;
+  opacity: 0.65;
 }
 
 .filterNavigatorBarTransition-enter.filterNavigatorBarTransition-enter-active {
+  color: var(--internal-selected-color);
   opacity: 1;
-  transform: translateX(0);
-  transition: opacity 300ms ease-out, transform 300ms ease-out;
+  transition: color 300ms, opacity 300ms;
 }
 
 .filterNavigatorBarTransition-exit {
   opacity: 1;
-  transform: translateX(0);
 }
 
 .filterNavigatorBarTransition-exit.filterNavigatorBarTransition-exit-active {
```

Comment:
@julienw n.b. This is another style for modifying style similar to your `className__property` in #3029.

Reply:
Yeah, but that one is imposed by react-transition-library.